### PR TITLE
Changelog for V8

### PR DESCRIPTION
**Pull request by glnro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,9 +34,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (gaia) Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.45.11](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.11). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for details.
+* (gaia) Bump [tendermint](https://github.com/tendermint/tendermint) to [0.34.24](https://github.com/tendermint/tendermint/tree/v0.34.24). See [CHANGELOG.md](https://github.com/tendermint/tendermint/blob/v0.34.24/CHANGELOG.md) for details.
+* (gaia) bump [iquidity](https://github.com/Gravity-Devs/liquidity) to [v1.5.3](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.5.3).
+* (gaia) bump [packet-forwarding-middleware](https://github.com/strangelove-ventures/packet-forward-middleware) to [v3.0.0](https://github.com/strangelove-ventures/packet-forward-middleware/releases/tag/v3.0.0).
+* (feat) Add [globalfee](https://github.com/cosmos/gaia/tree/main/x/globalfee) module.
```
Comment (yaruwangway) [inline CHANGELOG.md]: please add the docs here for global fee, https://github.com/cosmos/hub-eng/blob/main/tools/testing/scripts/simulations/globalfee/test_plan.md#if-globalfee-change-to-be-higher-than-min_gas_prices-what-is-the-influence-on-hermes-dev-testnet-and-testnet-might-have-this-test-case

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,9 +34,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (feat) [#1845](https://github.com/cosmos/gaia/pull/1845) Add bech32-convert command to gaiad.
+* (fix) [Adds new fee decorator](https://github.com/cosmos/gaia/pull/1961) to change `MaxBypassMinFeeMsgGasUsage` so importers of x/globalfee can change `MaxGas`.
```
Comment (yaruwangway) [inline CHANGELOG.md]: this one should stay together with global fee ? 
 i dont think we can change MaxGas, it is hard-coded now

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,9 +34,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (fix) [#1924](https://github.com/cosmos/gaia/issues/1924) Refactor iterator closures.
```
Comment (yaruwangway) [inline CHANGELOG.md]: do we include minor refactors to changelog ? @mmulji-ic

File: CHANGELOG.md [hunk 4]
```diff
@@ -34,9 +34,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (fix) [#1870](https://github.com/cosmos/gaia/issues/1870) Fix bank denom metadata in migration. See [#1892](https://github.com/cosmos/gaia/pull/1892) for more details.
+* (fix) [#1976](https://github.com/cosmos/gaia/pull/1976) Fix Quicksilver ICA exploit in migration.
```
Comment (yaruwangway) [inline CHANGELOG.md]: if there is some issue description, can you link here.

File: CHANGELOG.md [hunk 5]
```diff
@@ -34,9 +34,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (tests) Add [E2E tests](https://github.com/cosmos/gaia/tree/main/tests/e2e). The tests cover transactions/queries tests of different modules, including Bank, Distribution, Encode, Evidence, FeeGrant, Global Fee, Gov, IBC, ICA, packet forwarding middleware, Slashing, Staking, and Vesting module.
```
Comment (yaruwangway) [inline CHANGELOG.md]: ica  e2e tests are removed

File: CHANGELOG.md (hunk 2 above)
Comment (glnro) [reply CHANGELOG.md]: This is detailing the work done in https://github.com/cosmos/gaia/pull/1961

File: CHANGELOG.md (hunk 4 above)
Comment (glnro) [reply CHANGELOG.md]: There is no issue but we can link the forum post https://forum.cosmos.network/t/upcoming-interchain-accounts-bugfix-release/8911

File: CHANGELOG.md (hunk 1 above)
Comment (glnro) [reply CHANGELOG.md]: Those docs aren't publicly accesible. Going to update with https://github.com/cosmos/gaia/blob/main/docs/modules/globalfee.md

File: CHANGELOG.md (hunk 4 above)
Comment (yaruwangway) [reply CHANGELOG.md]: yeah, lets add that

File: CHANGELOG.md [hunk 6]
```diff
@@ -34,9 +34,25 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [v8.0.0] - 2023-1-05
+
+* (gaia) Bump [ibc-go](https://github.com/cosmos/ibc-go) to [v3.4.0](https://github.com/cosmos/ibc-go/blob/v3.4.0/CHANGELOG.md) to fix a vulnerability in ICA. See [v3.4.0 CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/v0.45.9/CHANGELOG.md) and [v3.2.1 Release Notes](https://github.com/cosmos/ibc-go/releases/tag/v3.2.1) for details.
+* (gaia) Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.45.11](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.11). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for details.
+* (gaia) Bump [tendermint](https://github.com/tendermint/tendermint) to [0.34.24](https://github.com/tendermint/tendermint/tree/v0.34.24). See [CHANGELOG.md](https://github.com/tendermint/tendermint/blob/v0.34.24/CHANGELOG.md) for details.
+* (gaia) bump [iquidity](https://github.com/Gravity-Devs/liquidity) to [v1.5.3](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.5.3).
```
Comment (yaruwangway) [inline CHANGELOG.md]: bump -> Bump

File: CHANGELOG.md [hunk 7]
```diff
@@ -34,9 +34,25 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+* (gaia) bump [packet-forwarding-middleware](https://github.com/strangelove-ventures/packet-forward-middleware) to [v3.0.0](https://github.com/strangelove-ventures/packet-forward-middleware/releases/tag/v3.0.0).
+* (feat) Add [globalfee](https://github.com/cosmos/gaia/tree/main/x/globalfee) module. See [globalfee docs](https://github.com/cosmos/gaia/blob/main/docs/modules/globalfee.md) for more details.
+* (feat) [#1845](https://github.com/cosmos/gaia/pull/1845) Add bech32-convert command to gaiad.
+* (fix) [Adds new fee decorator](https://github.com/cosmos/gaia/pull/1961) to change `MaxBypassMinFeeMsgGasUsage` so importers of x/globalfee can change `MaxGas`.
```
Comment (yaruwangway) [inline CHANGELOG.md]: Adds -> Add

**Review by yaruwangway — APPROVED** [review]

**Pull request merged**